Add Skinning Contest: Icons of an Era wiki page (#9339)
ppy/osu-wiki: wiki/Contests/Skinning_Contest/4/en.md

---
tags:
  - skinning
  - skinship
  - skins
---

# Skinning Contest #4

![Skinning Contest #4 banner](img/banner.jpg)

The **Skinning Contest #4: Chromatic Alteration** is a free-for-all osu! skinning contest hosted by [skinship](https://skinship.xyz), one of the largest skinning communities on osu!. It is the third official osu! skinning contest.

## Contest schedule

| Event | Timestamp |
| --: | :-- |
| Announcement | 2023-05-10 |
| Submission phase | 2023-05-10/2023-07-14 |
| Voting phase | 2 weeks starting from a separate news post |

> | Results | TBD |

Reviewer: ```suggestion
| Results | *TBD* |
```

## Prizes

| Placing | Prize(s) |
| :-: | :-- |
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter, unique profile badge |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 4 months of osu!supporter, unique profile badge |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 2 months of osu!supporter, unique profile badge |

## Organisation

The Skinning Contest #4 is run by various community members.

| User | Responsibility |
| :-- | :-- |

> | ::{ flag="RS" }:: [0x84f](https://osu.ppy.sh/users/7944724) | Newspost writer |
> | ::{ flag="DE" }:: [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
> | ::{ flag="RO" }:: [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning |
> | ::{ flag="NL" }:: [Roan](https://osu.ppy.sh/users/8214639) | Contest planning, internal tool development |
> | ::{ flag="DE" }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening |
> | ::{ flag="GB" }:: [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning |
> | ::{ flag="NL" }:: [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
> | ::{ flag="PL" }:: [watterino](https://osu.ppy.sh/users/3512261) | Video editor |
> | ::{ flag="TR" }:: [Zeus-](https://osu.ppy.sh/users/5464437) | Contest planning |

Reviewer: ```suggestion
| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | News post writing |
| ::{ flag=DE }:: [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
| ::{ flag=RO }:: [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning |
| ::{ flag=NL }:: [Roan](https://osu.ppy.sh/users/8214639) | Contest planning, internal tool development |
| ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening |
| ::{ flag=GB }:: [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning |
| ::{ flag=NL }:: [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
| ::{ flag=PL }:: [watterino](https://osu.ppy.sh/users/3512261) | Video editing |
| ::{ flag=TR }:: [Zeus-](https://osu.ppy.sh/users/5464437) | Contest planning |
```

> | ::{ flag="RS" }:: [0x84f](https://osu.ppy.sh/users/7944724) | Newspost writer |
> | ::{ flag="DE" }:: [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
> | ::{ flag="RO" }:: [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning |
> | ::{ flag="NL" }:: [Roan](https://osu.ppy.sh/users/8214639) | Contest planning, internal tool development |
> | ::{ flag="DE" }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening |
> | ::{ flag="GB" }:: [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning |
> | ::{ flag="NL" }:: [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
> | ::{ flag="PL" }:: [watterino](https://osu.ppy.sh/users/3512261) | Video editor |
> | ::{ flag="TR" }:: [Zeus-](https://osu.ppy.sh/users/5464437) | Contest planning |

Reviewer: ```suggestion
| ::{ flag=RS }:: [0x84f](https://osu.ppy.sh/users/7944724) | News post writing |
| ::{ flag=DE }:: [Master-TigerKun](https://osu.ppy.sh/users/10688456) | Contest planning |
| ::{ flag=RO }:: [Matt2e2](https://osu.ppy.sh/users/12144912) | Contest planning |
| ::{ flag=NL }:: [Roan](https://osu.ppy.sh/users/8214639) | Contest planning, internal tool development |
| ::{ flag=DE }:: [RockRoller](https://osu.ppy.sh/users/8388854) | Contest planning, submission screening |
| ::{ flag=GB }:: [tetsui](https://osu.ppy.sh/users/10974678) | Contest planning |
| ::{ flag=NL }:: [vvivi](https://osu.ppy.sh/users/10432755) | Contest planning |
| ::{ flag=PL }:: [watterino](https://osu.ppy.sh/users/3512261) | Video editing |
| ::{ flag=TR }:: [Zeus-](https://osu.ppy.sh/users/5464437) | Contest planning |
```

  - ranking panel
  - song selection
  - mode selection
  - mod icons
  - pause and fail menu
  - main menu (optional, but highly encouraged)
- Submissions must be new creations — only skins posted within the submission period will be accepted.
- Each submission must be accompanied by a forum thread in order to participate in the contest.
- Submissions must be made in time. Latecomers will not be accepted.
- Submissions must not be created as a part of, or be involved in a paid commission of any sort.
- Submissions must be created individually. Teams/collaborations are not allowed.
- Submissions must be in line with the given theme: "Icons of an Era".
- The file size for your `.osk` submission must be below 100 MB.
- Please submit a 16:9 image representing your skin, since this will be used as a cover on the contest page. This image must be at least 1280x720.

**Failure to comply with the contest rules may lead to disqualification, or, in severe cases, exclusion from future contests.**

## Theme: Icons of an Era {#theme}

> This iteration's theme, **Icons of an Era**, aims to spark the community's imagination by creating art around various iconic osu! beatmaps that were crafted and risen to glory throughout osu!'s lifetime, as well as the many either retired, official or community-made osu! mascots that have been introduced over the course of the past 15 years.
> 
> Whether your entry is focused around pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), a group of beatmap backgrounds from a specific era in osu! history, or just a skin focused around objectively the best mascot that has ever grazed osu! (Taikonator 4000), you will need to make sure that the theme is prominent as anyone should be able to open your entry and almost immediately recognise what the inspiration behind your entry was.

Reviewer: ```suggestion
This iteration's theme, **Icons of an Era**, aims to spark the community's imagination by creating art around various iconic osu! beatmaps that were crafted and risen to glory throughout osu!'s lifetime, as well as the many either retired, official or community-made osu! mascots that have been introduced over the course of the past 15 years.

Each entry should have a prominent theme where the inspiration behind it is immediately recognisable, such as:

- pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891)
- A group of beatmap backgrounds from a specific era in osu! history
- The best mascot that has ever grazed osu! ([Taikonator 3000](/wiki/Mascots#taikonator))
```

> Whether your entry is focused around pippi singing into the microphone from [The Big Black](https://osu.ppy.sh/beatmapsets/41823#osu/131891), a group of beatmap backgrounds from a specific era in osu! history, or just a skin focused around objectively the best mascot that has ever grazed osu! (Taikonator 4000), you will need to make sure that the theme is prominent as anyone should be able to open your entry and almost immediately recognise what the inspiration behind your entry was.

Reviewer: the wiki uses neutral writing and a passive voice, so this paragraph should be removed or rewritten

Author: Im open for suggestions as I simply copy pasted this from the newspost and have no better idea how to write it
